assets/js/components/IngestSheet/Completed.jsx

```diff
@@ -29,6 +31,37 @@ const IngestSheetCompleted = ({ sheetId, title }) => {
     data: errorsData,
   } = useQuery(INGEST_SHEET_COMPLETED_ERRORS, { variables: { id: sheetId } });
 
+  React.useEffect(() => {
+    const fn = async () => {
+      const response = await elasticsearchDirectSearch({
+        query: {
+          bool: {
+            must: [
+              {
+                match: {
+                  "model.name.keyword": "Image",
+                },
+              },
+              {
+                match: {
+                  "model.application": "Meadow",
+                },
+              },
+              {
+                match: {
+                  "sheet.id": sheetId,
+                },
+              },
+            ],
+          },
+        },
+        size: 0,
+      });
+      setTotalWorks(response.hits.total);
+    };
+    //fn();
+  }, [sheetId]);
+
```

Comment:
@kdid This is where we're now grabbing the info direct from Elasticsearch.

Reply:
Is it working for you locally? I'm seeing your getting results from Elasticsearch but I'm not seeing it show in the UI.

Reply:
also `yarn.lock` conflict

Reply:
@kdid Just fixed up both now.  I think I messed up a local stash before the last push, or something weird.  Everything should display now and blew away the `yarn.lock` for a fresh copy.